falcon/request.py

```diff
@@ -1148,26 +1148,26 @@ def get_param_as_list(self, name,
 
         raise errors.HTTPMissingParam(name)
 
-    def get_param_as_date(self, name, format_string='%Y-%m-%d',
-                          required=False, store=None):
-        """Return the value of a query string parameter as a date.
+    def get_param_as_datetime(self, name, format_string='%Y-%m-%dT%H:%M:%S',
```

Comment:
Upon further reflection, I think it may be better to require the `Z` UTC timezone designator (i.e., `%Y-%m-%dT%H:%M:%SZ`)  in the default format. The above format string implies ISO 8601, in which case the absence of a time zone designator indicates local time, which is ambiguous and therefore something that we should not encourage.